src/views/search_suggestions/search_suggestions_top_view.js

```diff
@@ -0,0 +1,20 @@
+import BaseView from '../base_view';
```

Comment:
There's a lot of boilerplate in these views. Can't help but feel like we could do better using inheritance.